README.md

```diff
@@ -120,7 +104,7 @@ To deploy OPA-Istio include the following container in your Kubernetes Deploymen
 
 ```yaml
 containers:
-- image: openpolicyagent/opa:0.9.0-istio
+- image: openpolicyagent/opa:0.9.0-istio-4
```

Comment:
What is the format for the image name ?

Reply:
The format is `<opa-version>-istio[-<commits-since-last-revendor>]`.

Since there are 4 commits since the last revendor `-4` is appended to the version.

When we eventually merge the `v0.9.1` update into the repo, it'll go back to `0.9.1-istio`.